product/01-introduction.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the learning path section for product managers.
+After watching this segment, you will have a better understanding of how InnerSource relates to Agile development best practices.
+While we all strive for autonomous teams that can move in an independent way, it is clear that in an complex, interconnected world some dependencies cannot be avoided.
+InnerSource provides an alternative to "wait it out", "build workarounds" and "escalate": Teams that need modifications in their dependencies can themselves offer a helping hand.
+InnerSource facilitates cross team collaboration.
+Through it's focus on written communication, InnerSource values are particularly helpful for teams moving to a remote first mode.
+
+In this Learning Path segment, you will learn where Agile development and InnerSource use similar terminology and even technology - but differ substantially in the details.
```

Comment:
```suggestion
In this Learning Path section, you will learn where Agile development and InnerSource use similar terminology and even technology - but differ substantially in the details.
```

Use "Section" (not "Segment") here.  See [Content Organization](https://github.com/InnerSourceCommons/InnerSourceLearningPath/#content-organization).